src/test/java/HTMLTests.java

```diff
@@ -0,0 +1,64 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+
+public class HTMLTests {
+    private WebDriver driver;
+
+    @BeforeClass
+    public void setProp() {
+        System.setProperty("webdriver.chrome.driver", "src/test/java/resources/chromedriver");
+    }
+
+    @BeforeMethod
+    public void setUp() {
+        driver = new ChromeDriver();
+        driver.get("http://localhost:63342/BodyMassIndexCalc/resources/index.html?_ijt=o5b07lt9h164bved46d4krv18s&_ij_reload");
+    }
+
+    @Test
+    public void validateWelcomeMessageTest() {
+       Assert.assertTrue(driver.findElement(By.id("header")).isDisplayed());
+    }
+    @Test
+    public void validateSelectAuthors() {
```

Comment:
Let's follow one approach in naming of methods

Reply:
renamed methods

Reply:
@KarynaMashkanava now you have 2 different approaches:
![Screenshot_4](https://user-images.githubusercontent.com/13552476/152933140-04b12139-7279-465a-9f7f-828fe316d100.png)
With 'Test' on the end is preferable, let's follow one approach